doc/source/plotting.rst

```diff
@@ -116,6 +116,34 @@ An example of plotting 2 DOS on different axes on the same figure:
   plot_1d_to_axis(dos2, axes[1])
   fig.show()
 
+Styling
```

Comment:
Sorry to nitpick, but could this heading be on a higher level like
```
Styling
=======
```
Currently its showing under the 1D Plots heading when really it applies to any kind of plot.

Reply:
Where in the heading order would you like it?

Reply:
After the final docstrings (2D) heading?

Reply:
Done! A bit annoying that it doesn't indent the Contents section appropriately.